src/drive/web/modules/upload/index.js

```diff
@@ -36,8 +38,11 @@ const status = action => {
   }
 }
 
-const item = (state, action) =>
-  Object.assign({}, state, { status: status(action) })
+const item = (state, action = { isUpdate: false }) =>
+  Object.assign({}, state, {
```

Comment:
Nit, but we don't use `Object.assign` anymore, we can use object destructuring instead.